lib/call-session.js

```diff
@@ -611,7 +611,8 @@ Duration=${payload.duration} `
         direction,
         sdp: offeredSdp,
       };
-      if (reason && opts.flags && !opts.flags.includes('reset')) opts.flags.push('reset');
+      // Dont reset ICE
```

Comment:
can you just update the comment so if we circle back to this months from now we'll remember
```
// Dont reset ICE - causes audiocodes webrtrc to fail with "missing ice-ufrag and ice-pwd in re-invite"